x/auth/stdtx.go

```diff
@@ -55,28 +55,21 @@ func (tx StdTx) ValidateBasic() sdk.Error {
 	if len(stdSigs) != len(tx.GetSigners()) {
 		return sdk.ErrUnauthorized("wrong number of signers")
 	}
-	if len(tx.GetMemo()) > maxMemoCharacters {
-		return sdk.ErrMemoTooLarge(
-			fmt.Sprintf(
-				"maximum number of characters is %d but received %d characters",
-				maxMemoCharacters, len(tx.GetMemo()),
-			),
-		)
-	}
 
 	sigCount := 0
 	for i := 0; i < len(stdSigs); i++ {
 		sigCount += countSubKeys(stdSigs[i].PubKey)
-		if sigCount > txSigLimit {
+		if uint64(sigCount) > DefaultTxSigLimit {
```

Comment:
Very strange. It is a param. Must've forgotten to update this.

Reply:
Ohhh wait, this is because it's in `ValidateBasic` too. I don't know of a clean way really to support this. Maybe `ValidateBasic` can take a `params interface{}...`?

Reply:
I guess this is OK for now. Maybe let's discuss in an issue

Reply:
Created an issue here: https://github.com/cosmos/cosmos-sdk/issues/3164